### PR TITLE
MINOR: KAFKA-3373 follow-up, a few val renames remaining

### PR DESCRIPTION
**Pull request by ijuma**
I also slightly tweaked the wording on a couple of warnings.

### CODE REVIEW

**Comment by ijuma** [comment]
@gwenshap, maybe you can review this.

File: core/src/main/scala/kafka/log/LogConfig.scala [hunk 1]
```diff
@@ -137,9 +137,9 @@ object LogConfig {
     "standard compression codecs ('gzip', 'snappy', lz4). It additionally accepts 'uncompressed' which is equivalent to " +
     "no compression; and 'producer' which means retain the original compression codec set by the producer."
   val PreAllocateEnableDoc ="Should pre allocate file when create new segment?"
-  val MessageFormatVersionDoc = KafkaConfig.MessageFormatVersionDoc
-  val MessageTimestampTypeDoc = KafkaConfig.MessageTimestampTypeDoc
-  val MessageTimestampDifferenceMaxMsDoc = KafkaConfig.MessageTimestampDifferenceMaxMsDoc
+  val MessageFormatVersionDoc = KafkaConfig.LogMessageFormatVersionDoc
```
Comment (gwenshap) [inline core/src/main/scala/kafka/log/LogConfig.scala:140]: The naming was intentional - everything in LogConfig is a log, so we don't bother to use it in the prop or variable names.

Comment (ijuma) [inline core/src/main/scala/kafka/log/LogConfig.scala:140]: I am not sure I understand. I didn't rename the LogConfig variables, I renamed the ones in KafkaConfig.

**Comment by gwenshap** [comment]
sorry, my misread. LGTM.

**Pull request closed without merging**